pkg/provisioner/core/provisioner.go

```diff
@@ -34,10 +34,15 @@ import (
 	"github.com/oracle/oci-volume-provisioner/pkg/oci/client"
 	"github.com/oracle/oci-volume-provisioner/pkg/oci/instancemeta"
 	"github.com/oracle/oci-volume-provisioner/pkg/provisioner/block"
+	"github.com/oracle/oci-volume-provisioner/pkg/provisioner/filestorage"
 	"github.com/oracle/oci-volume-provisioner/pkg/provisioner/plugin"
 )
 
 const (
+	// ProvisionerNameBlock name to deploy block provisioner by
+	ProvisionerNameBlock = "oracle.com/oci"
+	// ProvisionerNameFss name to deploy block provisioner by
```

Comment:
ProvisionerNameFss is the name of the OCI FSS dedicated storage provisioner

Reply:
Done